src/sentry/testutils/fixtures.py

```diff
@@ -627,6 +627,15 @@ def create_request_access(self, *args, **kwargs):
     def create_webhook_payload(self, *args, **kwargs):
         return Factories.create_webhook_payload(*args, **kwargs)
 
+    def create_dashboard(self, *args, **kwargs):
+        return Factories.create_dashboard(*args, **kwargs)
+
+    def create_dashboard_widget(self, *args, **kwargs):
+        return Factories.create_dashboard_widget(*args, **kwargs)
+
+    def create_dashboard_widget_query(self, *args, **kwargs):
+        return Factories.create_dashboard_widget_query(*args, **kwargs)
```

Comment:
these are great! 😍 Can we create a followup ticket to use them in other tests? AFAIK there are a couple of helpers that create widgets and dashboards that could all be streamlined to this.

Reply:
I was planning to do it, I have it on my refactor TODO list :)